src/apps/leaderboards/strategies.py

```diff
@@ -0,0 +1,123 @@
+import logging
```

Comment:
@ckcollab Do you know why Github thinks this is a new file here?

Reply:
This already exists on develop.

Reply:
Looks like I just needed to merge develop in again. Looks fine now.